tests/Exception/ComponentExceptionTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Unit\Exception;
+
+use EightshiftLibs\Exception\ComponentException;
+use stdClass;
+
+use function Tests\setupMocks;
+
+beforeAll(function () {
+	setupMocks();
+});
+
+test('Throws exception if ensure_string argument is invalid.', function () {
+
+	$object = new stdClass();
+	$integer = 7;
+	$null = null;
+	$bool = true;
+
+	$exceptionObject = ComponentException::throwNotStringOrArray($object);
+	$exceptionInteger = ComponentException::throwNotStringOrArray($integer);
+	$exceptionNull = ComponentException::throwNotStringOrArray($null);
+	$exceptionBool = ComponentException::throwNotStringOrArray($bool);
+
+	$this->assertIsObject($exceptionObject);
+	$this->assertIsObject($exceptionInteger);
+	$this->assertIsObject($exceptionNull);
+	$this->assertIsObject($exceptionBool);
+	
+	$this->assertObjectHasAttribute('message', $exceptionObject);
+	$this->assertObjectHasAttribute('message', $exceptionInteger);
+	$this->assertObjectHasAttribute('message', $exceptionNull);
+	$this->assertObjectHasAttribute('message', $exceptionBool);
+
+	$this->assertStringContainsString('Object couldn&#039;t be converted to string. Please provide only string or array.', $exceptionObject->getMessage());
```

Comment:
```suggestion
	$this->assertStringContainsString('Object couldn\'t be converted to string. Please provide only string or array.', $exceptionObject->getMessage());
```

Just escape the `'` character if you are using it inside a string 😉